airbyte-integrations/connectors/source-shopify/source_shopify/schemas/abandoned_checkouts.json

```diff
@@ -574,7 +574,7 @@
                 "type": ["null", "integer"]
               },
               "default": {
-                "type": ["null", "boolean"]
+                "type": ["null", "string"]
```

Comment:
wouldn't it be better to return `null` for these fields instead of changing its value for all customers to be string?

Reply:
It was a part of the test, accidentally pushed this change here. Fixed.